Cargo.toml

```
@@ -294,7 +294,7 @@ flate2 = "1.0"
 serde = { version = "1", features = ["derive"] }
 bytemuck = "1.7"
 # Needed to poll Task examples
-futures-lite = "1.11.3"
+futures-lite = { version = "2.0.1" }
```

should probably remove the `version` for consistency.
 
```suggestion
futures-lite = "2.0.1"
```